kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ResourceSelection.vue

```diff
@@ -1,20 +1,521 @@
 <template>
 
-  <KTextbox ref="textbox" label="Hit TAB, focus should go here" style="margin-top: 2em;" />
+  <div class="select-resource">
+    <div v-if="loading">
+      <KCircularLoader />
+    </div>
+    <div v-else>
+      <h5
+        class="title-style"
+      >
+        <KRouterLink
+          :to="goBack"
+        >
+          <KIcon
+            icon="back"
+          />
+        </KRouterLink>
+        {{ /* selectFoldersOrExercises$() */ }}
+      </h5>
+
+      <div v-if="!isTopicIdSet && bookmarks.length">
+
+        <p>{{ selectFromBookmarks$() }}</p>
+
+        <div @click="lessonCardClicked">
+          <KRouterLink
+            :appearanceOverrides="{
+              width: '100%',
+              textDecoration: 'none',
+              color: $themeTokens.text
+            }"
+            :to="getBookmarksLink"
+          >
+            <div :class="windowIsSmall ? 'mobile-bookmark-container' : 'bookmark-container'">
+              <BookmarkIcon :class="windowIsSmall ? 'mobile-bookmark-icon' : ''" />
+              <div :class="windowIsSmall ? 'mobile-text' : 'text'">
+                <h3>{{ coreString('bookmarksLabel') }}</h3>
+                <p>{{ numberOfSelectedBookmarks$({ count: bookmarks.length }) }}</p>
+              </div>
+            </div>
+          </KRouterLink>
+        </div>
+      </div>
+
+      <!-- <LessonsSearchFilters
```

Comment:
Is this intended to be integrated? Presumably it would only be keyword search at the moment?

Reply:
Initially I suspected we might keep some of the search-related logic in place, but as we will be revamping the coach search experience altogether to make use of existing search business from Learn, I'll just remove old search stuff altogether. It can just be referenced by looking at a <=release-v0.16.x branch in the future if needed.

Reply:
Discussed on slack - we'll file a follow up issue to restore keyword search on top of this implementation, but we'll make it non-blocking for release.